.gitignore

```diff
@@ -1,3 +1,14 @@
+# User-specific files
+*.suo
+*.user
+*.idea
+*.sln.docstates
+.vs/
+*.lock.json
+developer/
+launch.json
+launchSettings.json
```

Comment:
I don't think we want to get rid of this one we use this so everyone has F5 debugger